.github/workflows/ci.yml

```diff
@@ -105,7 +98,7 @@ jobs:
     strategy:
       max-parallel: 5
```

Comment:
Should this be changed to 4, like [23](https://github.com/tryolabs/norfair/blob/0befebc9b21e7914387e8fce9a610ef5001adf2f/.github/workflows/ci.yml#L23)?